ark/segmentation/marker_quantification.py

```diff
@@ -108,14 +111,17 @@ def generate_expression_matrix(segmentation_labels, image_data, nuclear_counts=F
     """Create a matrix of cells by channels with the total counts of each marker in each cell.
 
     Args:
-        segmentation_labels (xarray): xarray of shape [fovs, rows, cols, compartment] containing
-            segmentation masks for each FOV, potentially across multiple cell compartments
-        image_data (xarray): xarray containing all of the channel data across all FOVs
-        nuclear_counts (bool): boolean flag to determine whether nuclear counts are returned
+        segmentation_labels (xarray.DataArray):
+            xarray of shape [fovs, rows, cols, compartment] containing segmentation masks for each
+            FOV, potentially across multiple cell compartments
+        image_data (xarray.DataArray):
+            xarray containing all of the channel data across all FOVs
+        nuclear_counts (bool):
+            boolean flag to determine whether nuclear counts are returned
 
     Returns:
-        normalized_data (pandas): marker counts per cell normalized by cell size
-        arcsinh_data (pandas): arcsinh transfomed marker counts per cell normalized by cell size
+        Tuple of marker counts per cell normalized and arcsinh transfomed marker counts per cell
```

Comment:
This is a bit hard to follow. I wonder if there's a better way we can describe tuple return data?